sdk/package.json

```diff
@@ -45,6 +45,7 @@
     "test:get-position-ticks": "mocha ./tests/get-position-ticks.test.ts -g get-position-ticks",
     "test:get-liquidity-ticks": "mocha ./tests/get-liquidity-ticks.test.ts -g get-liquidity-ticks",
     "test:get-tickmap": "mocha ./tests/get-tickmap.test.ts -g tickmap",
+    "test:limits": "mocha ./tests/limits.test.ts -g limits",
```

Comment:
It's worth changing this name to `contract-storage-limit` or similar because we'll have several tests related to limits, so it's worth specifying which limit this test refers to